### DISCUSSION TITLE
CT-e Manual de Orientação do Contribuinte - Visão Geral - Versão 4.0 - fevereiro/2023

### DISCUSSION THREAD

**Discussion by wmmendes** [wmixvideo/nfe#896] [Geral]
Bom dia. Alguem está implementado a CT-e 4.0. Grato

**Comment by felipebz** [comment]
Só pra registrar aqui, estou implementando o CT-e 4.00.

Apesar do prazo "apertado", nem todos os estados disponibilizaram os ambientes do CT-e 4.00 ainda (como o PR, por exemplo). A Nota Técnica 2023.001 v.1.03 (de 01/06) definiu um prazo para as UF disponibilizarem os serviços:

> **Observação**: As datas de implantação no ambiente de homologação poderão variar conforme o ambiente de autorização, mas como data limite o dia 12/06/2023. 
> A implantação no ambiente de produção deverá ser padronizada para todos autorizadores, com data prevista para 26/06, podendo ser postergada em revisão desta NT.
